config/personality/en.ini

```diff
@@ -25,3 +25,12 @@ apps_add =
   com.endlessnetwork.drawingtutorials
   com.endlessnetwork.htmltutorials
   com.endlessnetwork.sciencesnacks
+
+[endlesskey]
+collections =
+  artist
+  athlete
+  curious
+  explorer
+  inventor
+  scientist
```

Comment:
I think you want to also include `extras` or this won't work offline. The explore plugin will try to import all the thumbnails from nodes in the `extras` pack at runtime. See https://github.com/endlessm/kolibri-explore-plugin/blob/master/kolibri_explore_plugin/collectionviews.py#L43. I think it's fine to import whatever nodes are specified in it.

Reply:
Oh yeah, that's a good one. I pushed a change just now (75060ae1bd7b43b54b93e3dbeaf5319a126a76bb) getting it to download all thumbnails for all the included channels.

Alas, adding the `extras` collection is a bit problematic, since that manifest file includes a bunch of _content_ as well. But it probably shouldn't? I proposed https://github.com/endlessm/endless-key-collections/pull/37 just now to address that.

Reply:
The collections change was merged, released, and included in an org.endlessos.Key flathub release. So, I think this should be good now.